gluonnlp/model/block.py

```diff
@@ -89,3 +90,100 @@ def __init__(self, axis=-1, eps=1E-6, **kwargs):
     def hybrid_forward(self, F, x):  # pylint: disable=arguments-differ
         ret = F.broadcast_div(x, F.norm(x, axis=self._axis, keepdims=True) + self._eps)
         return ret
+
+
+class CharacterLevelCNNEmbedding(HybridBlock):
```

Comment:
The CNN embedding used in the Bidaf paper follows this paper: https://arxiv.org/pdf/1508.06615.pdf, which uses highway and allows to specify multiple filters according different filter sizes. The input could be `(seq_len, batch_size, embedding_size)` and the output tensor should be `(batch_size, output_size)`, where the output_size could be the max_pooled_dimension. I already have the CNN embedding implementation in this PR: https://github.com/dmlc/gluon-nlp/pull/129. I suggested you could remove this character embedding in this PR.